### DISCUSSION TITLE
Retrieved observations data diverges from noaa.gov historical data

### DISCUSSION THREAD

**Discussion by timuralp** [weather-gov/api#245] [Q&A]
I'm trying to retrieve observations and cannot reconcile the responses I get from https://api.weather.gov/stations/KSEA/observations?start=<start time> with the historical observation data from https://www.wrh.noaa.gov/mesowest/getobext.php?wfo=sew&sid=KSEA&num=72&raw=0

When I query the api.weather.gov link for either /latest, I only get an observation from 01/24/21 4:53 PM local time, but the NOAA link above reports many observations since then. When I query for observations since 01/25/21 00:00:00 PST, I get no results, but NOAA link reports multiple observations since midnight.

Does the NOAA link use a different API for retrieving observations from stations?

**Comment by JJKraw** [comment]
Note that api.weather.gov has been and is having some serious operational problems over the last day or two. See https://www.nco.ncep.noaa.gov/status/messages/ for updates. I'm seeing many cases where data isn't getting updated. I'd retry this once they have it all up and running again.

**Comment by timuralp** [comment]
Thanks for the pointer -- I did not find the status updates on my own. I'll close this issue and try again once the operational issues are resolved.
